dockerfiles/Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN apt-get -y install \
         telnet \
         lsb-release \
         npm \
+        dnsutils \
```

Comment:
Do we need to add this dependency in our -ops repository (Salt configuration) as well?

Reply:
dig seems to be available in the webs, guess as a dependency of another dep, or maybe in the base ubuntu image (docker uses a minimal ubuntu). But no harm in being explicit I guess.